src/main/java/apoc/meta/Meta.java

```diff
@@ -234,12 +237,18 @@ public void rel(int typeId, String typeName, long count) {
             }
 
             public void rel(int typeId, String typeName, int labelId, String labelName, long out, long in) {
-                if (out > 0) relStats.put("(:" + labelName + ")-[:" + typeName + "]->()", out);
-                if (in > 0) relStats.put("()-[:" + typeName + "]->(:" + labelName + ")", in);
+                if (out > 0) {
+                    relStats.put("(:" + labelName + ")-[:" + typeName + "]->()", out);
+                    relStatsCount.put(typeName, out);
```

Comment:
is relstatscount global? if so shouldn't we add up the different counts per label?
actually there is a global count too of just rel-type in the count store